client/src/components/CreateMessagePanel/CreateMessagePanel.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const CreateMessagePanel = ({ sendMessage }) => {
+    const [userText, setUserText] = useState('');
+
+    const onSendClicked = () => {
+        if (userText.length > 0) {
```

Comment:
The message is coupled to text

Reply:
yeah for now, but bacuse im using websocket it's not a big deal to send, say blob (img for example).